### PR TITLE
Ido/add remove default container mode

### PR DESCRIPTION
**Pull request by IdoWeka**

### CODE REVIEW

File: topics/MBC_cluster_conversion/mbc_divider_script.py [hunk 1]
```diff
@@ -813,7 +815,9 @@ def main():
         host_remove_command = '/bin/sh -c "weka cluster host remove {} --no-unimprint"'.format(current_host_id)
         run_shell_command(host_remove_command)
         logger.info('the old container {} is disabled and stop, please remove it after the conversion is done'.format(container_name))
-
+        if args.remove_old_container:
+            container_rm_command = '/bin/sh -c "{}weka local rm {} -f"'.format(sudo, container_name)
```
Comment (danlweka) [inline topics/MBC_cluster_conversion/mbc_divider_script.py:819]: `{}weka` -> `{} weka`

File: topics/MBC_cluster_conversion/mbc_divider_script.py [hunk 2]
```diff
@@ -813,7 +815,9 @@ def main():
         host_remove_command = '/bin/sh -c "weka cluster host remove {} --no-unimprint"'.format(current_host_id)
         run_shell_command(host_remove_command)
         logger.info('the old container {} is disabled and stop, please remove it after the conversion is done'.format(container_name))
```
Comment (danlweka) [inline topics/MBC_cluster_conversion/mbc_divider_script.py:817]: nit: `stop` -> stopped

File: topics/MBC_cluster_conversion/mbc_divider_script.py (hunk 1 above)
Comment (danlweka) [reply topics/MBC_cluster_conversion/mbc_divider_script.py:819]: important!

**Review by danlweka — CHANGES_REQUESTED** [review]

File: topics/MBC_cluster_conversion/mbc_divider_script.py (hunk 1 above)
Comment (IdoWeka) [reply topics/MBC_cluster_conversion/mbc_divider_script.py:819]: actually no, the sudo include the sudo string include the whitespace

File: topics/MBC_cluster_conversion/mbc_divider_script.py (hunk 2 above)
Comment (IdoWeka) [reply topics/MBC_cluster_conversion/mbc_divider_script.py:817]: done

**Review by danlweka — APPROVED** [review]

**Pull request merged**